src/Chapter1.hs

```diff
@@ -635,8 +660,8 @@ Implement a function that returns the first digit of a given number.
 You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
-
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n = if div n 10 == 0 then n else firstDigit (div n 10)
```

Comment:
It looks like this function will produce wrong results on negative numbers, specifically, on numbers from `-1` to `-9`. And it probably will even hang on them...

Reply:
I think this is the reason of CI got canceled after 360m 😃